document/core/util/math.def

```diff
@@ -735,6 +735,9 @@
 .. |extend| mathdef:: \xref{exec/numerics}{op-extend_u}{\F{extend}}
 .. |extendu| mathdef:: \xref{exec/numerics}{op-extend_u}{\F{extend\_u}}
 .. |extends| mathdef:: \xref{exec/numerics}{op-extend_s}{\F{extend\_s}}
+.. |extend8s| mathdef:: \xref{exec/numerics}{op-extend8_s}{\F{extend8\_s}}
```

Comment:
These should also be named iextend for consistency (and nit, be moved up to the other i* macros).